dnf/cli/output.py

```diff
@@ -2151,12 +2151,7 @@ def error(self, message):
         pass
 
     def scriptout(self, msgs):
-        """Print messages originating from a package script.
-
-        :param msgs: the messages coming from the script
-        """
-        if msgs:
-            self.rpm_logger.info(ucd(msgs))
+        pass
```

Comment:
Doesn't this turn the logging off for this class, while it was logging messages here before?

Reply:
The whole logging part of dnf is kind of twisted, this is how it currently works:

- in Base.do_transaction() method is `LoggingTransactionDisplay()` always added to the displays list.
- `LoggingTransactionDisplay` is a parent class of `CliTransactionDisplay`. This display is used when dnf is invoked via command line interface.
- that means we have two displays enabled during transaction, one is a subclass of the other which results in double logging (see https://github.com/rpm-software-management/dnf/commit/657fd64942d35b5797cbc2ac5e780b9185eef99f, https://github.com/rpm-software-management/dnf/commit/dc277b657dd5f8f53c13af8606f2b886984ad412)

So this change makes scriptout() consistent with filelog() and error() methods, logging was moved to the parent class and needs to be switched off here to avoid double logging.